GoogleDataTransport.podspec

```diff
@@ -40,6 +40,7 @@ Shared library for iOS SDK data transport needs.
   s.libraries = ['z']
 
   s.dependency 'GoogleUtilities/Environment', '~> 7.2'
```

Comment:
Update to 7.7 as well. All deps to the same pod should match

Reply:
Done